Readme.md

TimurSirkin opened the pull request.

TimurSirkin committed:
Add PatchBuilder files and Templates.ini

TimurSirkin committed:
Create README.md

Nikolay2k commented on README.md:
```diff
@@ -0,0 +1,39 @@
+#### Входящие данные: 
+1) Список объектов, которые нужно собрать в патч. Список должен быть в формате .txt, где каждая строка - описание объекта в формате:
+`[схема] [имя] [тип] ( [название параметров через пробел] )`
+2) Параметры соединения с базой данных, из которой нужно получить объекты патча
```
нужны шаблон строки соединения и  пример соединения с БД

- [x] шаблон и пример

TimurSirkin replied:
Добавил шаблон, пример есть в конце файла

Nikolay2k commented on README.md:
```diff
@@ -0,0 +1,39 @@
+3) Дирректория, в которую нужно собрать все объекты патча
+4) [Не обязательно] Конфигурационный файл шаблонов, с помощью которого будут искаться зависимости в скриптах патча. Файл должен быть в формате .ini. 
+Стандартный файл шаблонов можно найти в репозитории
+#### Исходящие данные: 
```
Привычней **Выходные данные**

Nikolay2k commented on README.md:
```diff
@@ -0,0 +1,39 @@
+Стандартный файл шаблонов можно найти в репозитории
+#### Исходящие данные: 
+1) Набор скриптов, в которых описано создание объектов патча
+2) Список скриптов, которые нужно установить. Список должен быть в формате .txt
```
Вместо слова _нужно_ необходимо использовать слово _необходимо_, т.к. слово _нужно_ звучит более по бытовому

- [x] необходимо

TimurSirkin committed:
Changes required from pull request

Название -> наименование
Исходные данные -> выходные данные
Add connect arguments tamplate
